java/daikon/FileIO.java

```diff
@@ -1773,9 +1777,9 @@ public static void process_sample(
       //  and :::CLASS program points.  This scheme ensures that arbitrarly
       //  named program points such as :::POINT (used by convertcsv.pl)
       //  will be treated as leaves.
+      //  Throws is a LEAF now, like Exit_nn
```

Comment:
"now" isn't necessary.  Why is "LEAF" capitalized?
This line seems contradictory with line 1776.